### PR TITLE
Added Images and Inputs

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Images and Inputs in MADEUI.

### CODE REVIEW

File: components/Images/Images.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+.img_responsive{
```
Comment (sanathkumar26700) [inline components/Images/Images.css:2]: Image is _not responsive_ as you have specified _fixed_ length for width and height.
Try changing it into percentages hence the height and width will depend on its container  
**or**
 use media queries to change the height and width with respect to viewport.

**Review by manojasarsa — APPROVED** [review]
Looks good to me! Just a small suggestion : If you can give some top and bottom margin to input fields, it would look great!

File: components/Images/Images.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+     width:50rem;
+     height:30rem;
```
Comment (msdeshmukh009) [inline components/Images/Images.css:4]: Hey Shrinkhla actually the point of the responsive image is that the image will fit inside any container and take space according to the container's width which will make the image responsive. So giving the image fixed width and height might not help. Maybe try `width:100%; height:auto;` .

File: components/Images/Images.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+}
+.img_round{
+        width: 13rem;
+        height:13rem;
+        border-radius: 50%;
+    
+}
+.img_square{
+    width:14rem;
+    height:14rem;
```
Comment (msdeshmukh009) [inline components/Images/Images.css:14]: I think giving an image fixed width will be problematic from a reusability point of view. What you can do is create a `div` around the image and then give it sizes.

**Comment by Shrinkhlarajpoot** [comment]
> Looks good to me! Just a small suggestion : If you can give some top and bottom margin to input fields, it would look great!

Okay Thanks ,,Will surely look into this

File: components/Images/Images.css (hunk 2 above)
Comment (Shrinkhlarajpoot) [reply components/Images/Images.css:4]: thanks for clearing this..Will improve this.

File: components/Images/Images.css (hunk 3 above)
Comment (Shrinkhlarajpoot) [reply components/Images/Images.css:14]: Okay

**Pull request merged**